### DISCUSSION TITLE
Scheduled tasks/cron jobs

### DISCUSSION THREAD

**Discussion by sandervspl** [Dokploy/dokploy#615] [Ideas]
Coolify has this feature where you can add scheduled tasks to a service. You add a command, a cron string, and select the container that needs to run the task. Also shows a history of all runs and the output and success/error status.

**Comment by Siumauricio** [comment]
hi, This is something I plan to do together with the backups, as I want to generate the cron jobs/tasks issue into something unified without creating different interfaces.

**Reply by Code-Divine** [reply]
Is this developed now? I can't see this feature?

**Reply by Siumauricio** [reply]
Yes this is done a few months ago https://docs.dokploy.com/docs/core/schedule-jobs

**Reply by Code-Divine** [reply]
The way this is done it requires techical knowledge on the level this tool tries to encapsulate. The way it should work is select a container the scheduler should fire up, the container does the job and after that shuts down. We set up some variables for the container image and that's it.
